Tools/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "publisher",
+  "version": "0.1.0",
+  "description": "Publisher used for creating pdfs of mstg and masvs",
+  "main": "genpdf.js",
+  "scripts": {
+    "test": "-h"
+  },
+  "author": "Jeroen Willemsen",
+  "license": "MIT",
+  "dependencies": {
```

Comment:
Can you fix the coffee-script dependency? 

> npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)

Reply:
coffeescript is used by the dependencies, I guess the warning will live until we can update the dependencies itself not referencing coffee-script ;-)